lib/web/users.go

```diff
@@ -119,7 +119,29 @@ func updateUser(r *http.Request, m userAPIGetter, createdBy string) (*ui.User, e
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+
 	user.SetRoles(req.Roles)
+	if req.Logins != nil {
```

Comment:
Should we reuse this same block for createUser? Or is there a particular reason why we only support SetLogins there?

Reply:
We supported setting the logins before this PR
I only changed this because I changed the `saveUserRequest.Logins` from `[]string` into `*[]string`.

Besides that, the issue only mentioned the update path

I think we can add them when creating, so I'll extract the update traits section and re-use it when creating as well